gradle-plugin/src/main/java/com/contrastsecurity/gradle/plugin/ContrastVerifyTestTask.java

```diff
@@ -66,25 +66,20 @@ public void verifyFindings() {
     final String appId = getApplicationId(config, orgUuid);
     final String serverName = config.getServerName().get();
 
-    final List<String> serverIds = getServerId(sdk, appId, orgUuid, serverName);
+    final List<String> serverIds = getServerId(appId, orgUuid, serverName);
 
     final TraceFilterBody traceFilter =
         getTraceFilterBody(serverIds, config.getMinSeverity().get(), config.getAppVersion().get());
 
-    Traces traces;
-    try {
-      traces = sdk.getTraces(orgUuid, appId, traceFilter);
-    } catch (IOException e) {
-      throw new GradleException("Failed to retrieve traces from TeamServer", e);
-    }
+    final Traces traces = getTraces(orgUuid, appId, traceFilter);
 
     if (traces != null && traces.getCount() > 0) {
       logger.info(traces.getCount() + " new vulnerability(s) were found.");
 
       // Write TeamServer results to a file in the build directory
-      try (BufferedWriter writer =
+      try (final BufferedWriter writer =
```

Comment:
Consider using `Files.writeString(...)` or `Files.write(...)`, they're easier and more concise